message-index/messages/GHC-00482/example1/before/LambdaInCase.hs

```diff
@@ -0,0 +1,6 @@
+module LambdaInCase where
+
+f :: Int -> Int
+f x =
+  case x of
+    \a -> a
```

Comment:
I think that because the error specifically mentions *functions*, the example should be a case where they are trying to match on a function rather than an incorrect syntax. What do you think?

Reply:
Hmm, good point. I agree.

Reply:
I intended this message for the case of adding an accidental backslash. My other example is for the matching-on-function case (where the pattern is inline in a function definition), but I agree we could add another one.

Reply:
Yeah, "accidental backslash" is a good case to catch!